src/client/game/symbols.hpp

```diff
@@ -235,7 +235,7 @@ namespace game
 	WEAK symbol<const char*(scr_string_t stringValue)> SL_ConvertToString{0x3C0C50, 0x507CD0};
 	WEAK symbol<unsigned int(const char* str)> SL_GetCanonicalString{0x3BDA20, 0x504A00};
 
-	WEAK symbol<void(netadr_s* from)> SV_DirectConnect{0x0, 0x54DBF0};
+	WEAK symbol<void(netadr_s* from)> SV_DirectConnectExternal{0x0, 0x54DBF0};
```

Comment:
not sure why you renamed this?

Reply:
The symbol at that address is named `SV_DirectConnectExternal` as opposed to the norm of `SV_DirectConnect`. Renaming it was a small correction.

Reply:
SV_DirectConnectExternal is not a symbol im aware of, it isnt present in the iw6 or h1 pdbs

Reply:
That's odd - the IDA database given to me has this function labeled as `SV_DirectConnectExternal`. I'll revert the change.

Reply:
Ohhh I see. Thank you for clearing that up!